site/_docs/reference.md

```diff
@@ -2804,6 +2805,44 @@ To enable, include `calcite-server.jar` in your class path, and add
 to the JDBC connect string (see connect string property
 [parserFactory]({{ site.apiRoot }}/org/apache/calcite/config/CalciteConnectionProperty.html#PARSER_FACTORY)).
 
+
+
+#### Declaring Objects For Types Defined In Schema
+After an object type is defined and installed in the schema, you can use it to declare objects in any SQL block. For example, you can use the object type to specify the datatype of an attribute, column, variable, bind variable, record field, table element, formal parameter, or function result. At run time, instances of the object type are created; that is, objects of that type are instantiated. Each object can hold different values.
```

Comment:
Can you move it under `In createFunctionStatement and usingFile, classNameLiteral and filePathLiteral are character literals.` which is the end of the whole document.

Reply:
Done:)